src/guides/v2.4/graphql/interfaces/cart-item-interface.md

```diff
@@ -101,6 +101,7 @@ The `ConfigurableCartItem` object adds the following attributes to the `CartItem
 Attribute | Data type | Description
 --- | --- | ---
 `configurable_options` | [[SelectedConfigurableOption!]!](#SelectedConfigurableOption) | An array of configurable options
+`configured_variant` | [ProductInterface]({{page.baseurl}}/graphql/interfaces/product-interface.html) | Product details of the cart item
```

Comment:
there could be more to this:
 - regardless of the storeConfig value `configurable_thumbnail_source` which points to the child or parent while rendering cart the `configured_variant` will always show the child. It is up to the UI to query configured_variant when configurable_thumbnail_source is set to child. 
 There might other use cases when we want to know details about the `configured_variant` than just the thumbnail

Reply:
also FYI this PR that you're linking to is obsolete, there's a new PR https://github.com/magento/magento2/pull/31380